### PR TITLE
Support for multiple YB hosts and secondary index change

### PR DESCRIPTION
**Pull request by hbhanawat**
1. Added support to take the a comma separated list of hosts instead of a single host. This is needed to load balance between YB servers
2. Create secondary index before data loading

### CODE REVIEW

**Review by fritshoogland-yugabyte — COMMENTED** [review]
I do assume you're working on newer/latest builds of the database. 
Wouldn't it make much more sense to use a switch to spread over the available hosts which are pulled from yb_servers() (like the smart driver does)

**Comment by hbhanawat** [comment]
@fritshoogland-yugabyte At this point, I am not sure if I would like to limit sysbench to the latest YB branch. I would like to run tests on older branches as well. Hence, I would not jump to yb_servers immediately

File: src/drivers/pgsql/drv_pgsql.c [hunk 1]
```diff
@@ -112,6 +113,8 @@ static char use_ps; /* whether server-side prepared statemens should be used */
 
 /* PgSQL driver operations */
 
+
+static char** str_split(char *src, char sep, int *numparts);
```
Comment (suranjan) [inline src/drivers/pgsql/drv_pgsql.c:117]: Why not use strtok?

Comment (hbhanawat) [reply src/drivers/pgsql/drv_pgsql.c:117]: There are extra libraries required for this. I infact I used one of the library functions earlier and it started giving issues in ubuntu environment. Just to save ourselves the overhead of testing it with different environments, I moved away from library functions.  https://github.com/yugabyte/sysbench/pull/9/commits/c630e99da2a3d5ee2ab2ee4e702edc5df68db899

**Review by suranjan — APPROVED** [review]
Looks good.

**Pull request merged**